util/src/main/java/io/kubernetes/client/util/KubeConfig.java

```diff
@@ -395,7 +395,15 @@ private static String getData(Map<String, Object> obj, String key) {
     return null;
   }
 
-  public static byte[] getDataOrFile(final String data, final String file) throws IOException {
+  public byte[] getDataOrFileRelative(final String data, final String path) throws IOException {
```

Comment:
if this is an alternative for the old `getDataOrFile`, wdyt we also make this `public static`?

Reply:
It can't be static because it relies on the file location that the KubeConfig was loaded from.  I suppose we could make it static and pass in the file, but that seems unnecessarily complex.